internal/code-generator/generator/vendor.go

```diff
@@ -25,6 +26,12 @@ func Vendor(fileToParse, samplesDir, outPath, tmplPath, tmplName, commit string)
 		return fmt.Errorf("failed to parse YAML %s, %q", fileToParse, err)
 	}
 
+	for _, re := range regexps {
+		if !isRE2(re) {
+			log.Printf("RE2 incompatible syntax for vendor:'%s'\n", re)
```

Comment:
Would be nice if that was optional, example by global variable to enable logging (disabled by default) as this could potentially spam log files and user of this library could be using different logging library. Other option would be to set some logging interface to use as global variable. First option would probably be easier to do and enough at least for now

Reply:
Edit: I take my word back, this is for generator 😅

Reply:
Yes, indeed, these logs are only for code generation that is done before the release, to keep track of https://github.com/go-enry/go-enry#divergences-from-linguist.

It would be nice though to have a "debug mode" in the library, possibly, with instrumentation that will trace and log all the steps of decision process, like `Linguist.instrument` does.

Reply:
Yes instrumentation would definitely be way to go